bin/xam_coverage_bins.pl

```diff
@@ -25,23 +25,19 @@
 use Carp;
 use Const::Fast;
 
+use FindBin qw($Bin);
+use lib "$Bin/../lib";
```

Comment:
Without this the test's executing the script via shell don't work as the modules aren't installed.  Didn't break until it used modules in the package, but FYI